Softeq.NetKit.Components.EventBus.Service/EventBusService.cs

```diff
@@ -138,12 +142,16 @@ public void RegisterQueueListener(QueueListenerConfiguration configuration = nul
                 var handlerOptions = new SessionHandlerOptions(ExceptionReceivedHandler)
                 {
                     AutoComplete = false,
-                    MaxConcurrentSessions = configuration?.MaxConcurrent ?? 10
-
+                    MaxConcurrentSessions = configuration.MaxConcurrent
```

Comment:
That's actually impossible because this code relies on `useSessions` variable and if it's true than configuration is not null
```
var useSessions = configuration?.UseSessions ?? false;
if (useSessions) {...}
```